src/components/VerificationRequest/VerificationRequestDetailDrawer.tsx

```diff
@@ -0,0 +1,460 @@
+import React, { useEffect, useState } from "react";
+import { Grid, Typography, Box, Alert, AlertTitle } from "@mui/material";
+import {
+    WarningAmber,
+    Public,
+    DateRange,
+    Filter,
+    Share,
+    ArrowBackOutlined,
+} from "@mui/icons-material";
+import { useTranslation } from "next-i18next";
+import { useAtom } from "jotai";
+import verificationRequestApi from "../../api/verificationRequestApi";
+import { ReviewTaskTypeEnum } from "../../machines/reviewTask/enums";
+import { Roles } from "../../types/enums";
+import { currentUserRole } from "../../atoms/currentUser";
+import { useAppSelector } from "../../store/store";
+import colors from "../../styles/colors";
+import LargeDrawer from "../LargeDrawer";
+import Loading from "../Loading";
+import AletheiaButton, { ButtonType } from "../Button";
+import TopicDisplay from "../topics/TopicDisplay";
+import { MetaChip } from "./MetaChip";
+import { VerificationRequestContent } from "./VerificationRequestContent";
+import { RequestDates } from "./RequestDates";
+import SourceList from "./SourceListVerificationRequest";
+
+interface VerificationRequestDetailDrawerProps {
+    verificationRequest: any;
+    open: boolean;
+    onClose: () => void;
+    onUpdate?: () => void;
+}
+
+const getSeverityColor = (severity: string) => {
```

Comment:
suggestion: we use this same logic in different places, we could extract into a reusable and exportable function